pages/textbook-beta/course/multiple-qubits-and-entanglement.vue

```diff
@@ -0,0 +1,137 @@
+<script lang="ts">
+import { Course, Prerequisite } from 'constants/textbookBetaContent'
+import { Component } from 'vue-property-decorator'
+import CourseOverviewPage from '~/components/logic/CourseOverviewPage.vue'
+import { GeneralLink, MultipleQubitsAndEntanglementUrl } from '~/constants/appLinks'
+
+@Component({
+  head: () => ({
+    title: 'Quantum States and Qubits'
+  })
+})
+export default class MultipleQubitsAndEntanglementPage extends CourseOverviewPage {
+  routeName: string = 'multiple-qubits-and-entanglement'
+
+  headerTitle = 'Multiple Qubits and Entanglement'
+  headerDescription = ['We\'ve seen some interesting effects with isolated qubits and single qubit gates, but the true power of quantum computing comes from interactions between qubits. In this section we will introduce multiple qubit gates and explore the interesting behaviours of multi-qubit systems.']
+
+  headerImg = '/images/textbook-beta/course/multiple-qubits-and-entanglement/multiple-qubits-and-entanglement.png'
+
+  backToTextbookHomeLink: GeneralLink = {
+    url: '/textbook-beta',
+    label: 'Home',
+    segment: {
+      cta: 'back-to-textbook-home', location: 'header'
+    }
+  }
+
+  startLearningCTA: GeneralLink = {
+    url: MultipleQubitsAndEntanglementUrl.Introduction,
+    label: 'Start learning',
+    segment: {
+      cta: 'introduction', location: 'header'
+    }
+  }
+
+  references: string[] = []
+
+  links: GeneralLink[] = [
+    {
+      url: 'https://en.wikipedia.org/wiki/Quantum_Computation_and_Quantum_Information',
+      label: '(Chapter 1)',
+      segment: {
+        cta: 'quantum-computation-and-quantum-information',
+        location: 'external-recommended-readings'
+      }
+    },
+    {
+      url: 'https://en.wikipedia.org/wiki/Quantum_Computation_and_Quantum_Information',
+      label: '(Chapters 1-4)',
+      segment: {
+        cta: 'an-introduction-to-quantum-computing',
+        location: 'external-recommended-readings'
+      }
+    },
+    {
+      url: 'https://en.wikipedia.org/wiki/Quantum_Computation_and_Quantum_Information',
+      label: '(Chapters 1 and 2)',
+      segment: {
+        cta: 'quantum-computing-lecture-notes',
+        location: 'external-recommended-readings'
+      }
```

Comment:
all these recommended reading point to the same book/link. is this intentional?

Reply:
good catch! I don't think these are intended to be the same... 

will take another look and fix

Reply:
@vabarbosa - updated! 

BTW, as you know, we have a sync tomorrow morning to review the `ExternalRecommendedReadings` links and how we're going to structure the references... 

We can handle those updates in a separate PR, however they should change...

Reply:
yeah, i agree that can addressed in a separate PR